db-tests/src/test/java/TestTriggers.java

```diff
@@ -2,6 +2,7 @@
 import com.zaxxer.hikari.HikariConfig;
 import com.zaxxer.hikari.HikariDataSource;
 import org.junit.Assume;
+import org.apache.commons.lang3.tuple.Pair;
 import org.junit.jupiter.api.AfterAll;
```

Comment:
Imports are a little unordered here -- this `org.apache` is sandwiched between two `org.junit`.

Reply:
Addressed